### PR TITLE
Import v2

### PR DESCRIPTION
**Pull request by rmroot**
connects #146 

Import wizard for excel files
Updates template to excel with tabs/pages
New page for importing
Supports any excel file with column mapping for meters
Quick fix conversion bug in data tables

Still todo:
Predictors importing
Mapping entries by rows, map data fields by column

### CODE REVIEW

File: src/app/utility/upload-data/excel-data-table/excel-data-table.component.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.scroll-item{
```
Comment (cranfillkd) [inline src/app/utility/upload-data/excel-data-table/excel-data-table.component.css:1]: Duplicated in other css files. Move to global.css

File: src/app/utility/upload-data/excel-wizard/excel-wizard.component.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.scroll-item{
```
Comment (cranfillkd) [inline src/app/utility/upload-data/excel-wizard/excel-wizard.component.css:1]: Duplicated in other css files. Move to global.css

File: src/app/utility/upload-data/import-meter-data-wizard/invalid-meter-data-table/invalid-meter-data-table.component.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.header-width{
```
Comment (cranfillkd) [inline src/app/utility/upload-data/import-meter-data-wizard/invalid-meter-data-table/invalid-meter-data-table.component.css:1]: Duplicated in other css files. Move to global.css

File: src/app/utility/upload-data/import-meter-data-wizard/missing-meter-number-table/missing-meter-number-table.component.css [hunk 4]
```diff
@@ -0,0 +1,3 @@
+.header-width{
```
Comment (cranfillkd) [inline src/app/utility/upload-data/import-meter-data-wizard/missing-meter-number-table/missing-meter-number-table.component.css:1]: Duplicated in other css files. Move to global.css

File: src/app/utility/upload-data/meter-data-table/meter-data-table.component.css [hunk 5]
```diff
@@ -0,0 +1,7 @@
+.import-meter .file-name-width{
```
Comment (cranfillkd) [inline src/app/utility/upload-data/meter-data-table/meter-data-table.component.css:1]: Duplicated in other css files. Move to global.css

File: src/app/utility/upload-data/meter-table/meter-table.component.css [hunk 6]
```diff
@@ -0,0 +1,7 @@
+.import-meter .file-name-width{
```
Comment (cranfillkd) [inline src/app/utility/upload-data/meter-table/meter-table.component.css:1]: Duplicated in other css files. Move to global.css

File: src/app/utility/upload-data/upload-data.component.css [hunk 7]
```diff
@@ -0,0 +1,7 @@
+.import-meter .file-name-width{
```
Comment (cranfillkd) [inline src/app/utility/upload-data/upload-data.component.css:1]: Duplicated in other css files. Move to global.css

File: src/app/utility/upload-data/excel-wizard/setup-data-wizard/setup-data-wizard.component.css [hunk 8]
```diff
@@ -0,0 +1,8 @@
+.scroll-item{
```
Comment (cranfillkd) [inline src/app/utility/upload-data/excel-wizard/setup-data-wizard/setup-data-wizard.component.css:1]: Duplicated in other css files. Move to global.css

File: src/app/utility/upload-data/import-meter-data-wizard/import-meter-data-wizard.component.css [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+    margin-right: 10px;
+}
+
+
+.wizard-content{
+    max-height: 85%;
+    height: 85%;
+}
+
+.scroll-item{
```
Comment (cranfillkd) [inline src/app/utility/upload-data/import-meter-data-wizard/import-meter-data-wizard.component.css:15]: Duplicated in other css files. Move to global.css

File: src/app/utility/upload-data/import-meter-wizard/import-meter-wizard.component.css [hunk 10]
```diff
@@ -0,0 +1,7 @@
+.nav-link{
```
Comment (cranfillkd) [inline src/app/utility/upload-data/import-meter-wizard/import-meter-wizard.component.css:1]: Duplicated in other css files. Move to global.css

Comment (rmroot) [reply src/app/utility/upload-data/import-meter-wizard/import-meter-wizard.component.css:1]: The properties of the two nav-link classes are to set the font size and they are different sizes. Going to leave them

File: src/app/utility/upload-data/meter-data-table/meter-data-table.component.css (hunk 5 above)
Comment (rmroot) [reply src/app/utility/upload-data/meter-data-table/meter-data-table.component.css:1]: .import-meter .file-name is used in multiple places but isn't consistent in all the places. Going to leave there

File: src/app/utility/upload-data/upload-data.component.css (hunk 7 above)
Comment (rmroot) [reply src/app/utility/upload-data/upload-data.component.css:1]: .import-meter .file-name is used in multiple places but isn't consistent in all the places. Going to leave there

**Review by cranfillkd — APPROVED** [review]

**Pull request merged**